api/categorisation.go

```diff
@@ -0,0 +1,31 @@
+package api
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-cantabular-filter-flex-api/model"
+)
+
+/*
+   CheckDefaultCategorisation checks the default categorisation of a given dimension
+   so that we store the correct parent dimension for a given set of options?
+*/
+func (api *API) CheckDefaultCategorisation(dimName string) (dimension *model.Dimension, err error) {
+
+	ctx := context.Background()
+	cats, err := api.populations.GetCategorisations(ctx, nil)
+	names := make([]string, 0)
+
+	for _, cat := range cats {
+		names = append(names, cat.Name)
+	}
+	defaultCat, err := api.metadata.GetDefaultCategorisation(ctx, nil)
+
+	for _, categorisation := range defaultCat {
```

Comment:
not sure about this because in the unit tests of the metadata in api clients go, i saw that multiple categories could be default categorisations if I remember correctly